auto-value-gson/gradle.properties

```diff
@@ -0,0 +1,20 @@
+GROUP=com.ryanharter.auto.value
```

Comment:
This should remain at the root level, as these values will be shared among all modules (in case we ever add a processor/annotations module, etc.)